### PR TITLE
Feature multi seg wings

### PR DESCRIPTION
**Pull request by mclarke2**
This is a pull request for the Multi-Segment Wing Branch
It doesn't pass the aerodynamics regression (since the reference values are based on the simple trapezoidal wing of the B737). The results from this new Fidelity-Zero Aero model of the E190 aircraft (which doesn't  have segments) matches up with the old Fidelity-Zero Aero model

### CODE REVIEW

**Comment by planes** [comment]
Hey, looks like something broke on Travis. Take a look at the regressions.

File: regression/scripts/B737/mission_B737.py [hunk 1]
```diff
@@ -84,6 +84,8 @@ def main():
     old_results = load_results()   
 
     # plt the old results
+
```
Comment (WalterMaier) [inline regression/scripts/B737/mission_B737.py]: extra space?

File: regression/scripts/B737/mission_B737.py [hunk 2]
```diff
@@ -84,6 +84,8 @@ def main():
     old_results = load_results()   
 
     # plt the old results
+    
+    save_results(results)
```
Comment (WalterMaier) [inline regression/scripts/B737/mission_B737.py]: does this need to be saved every time?

File: regression/scripts/concorde/concorde.py [hunk 3]
```diff
@@ -63,7 +63,7 @@ def main():
     results = mission.evaluate()
     
     # load older results
-    #save_results(results)
+    save_results(results)
```
Comment (WalterMaier) [inline regression/scripts/concorde/concorde.py]: save everytime?

File: trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py [hunk 4]
```diff
@@ -321,10 +321,12 @@ def calculate_lift_vortex_lattice(conditions,settings,geometry):
...
     total_lift_coeff = 0.0
     wing_lifts = Data()
+
     for wing in geometry.wings.values():
 
         [wing_lift_coeff,wing_drag_coeff] = weissinger_vortex_lattice(conditions,settings,wing)
         total_lift_coeff += wing_lift_coeff * wing.areas.reference / vehicle_reference_area
         wing_lifts[wing.tag] = wing_lift_coeff
 
+
```
Comment (WalterMaier) [inline trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py]: extra space?

File: trunk/SUAVE/Components/Energy/Converters/__init__.py [hunk 5]
```diff
@@ -18,4 +18,4 @@
 from Turbine import Turbine
 from Supersonic_Nozzle import Supersonic_Nozzle
 from Shaft_Power_Off_Take import Shaft_Power_Off_Take
-from Gearbox import Gearbox
```
Comment (WalterMaier) [inline trunk/SUAVE/Components/Energy/Converters/__init__.py]: not used?

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py [hunk 6]
```diff
@@ -62,9 +64,9 @@ def weissinger_vortex_lattice(conditions,configuration,wing):
     sym_para    = wing.symmetric
     Sref        = wing.areas.reference
     orientation = wing.vertical
-
+    
     n  = configuration.number_panels_spanwise
-
+    n = 40
```
Comment (WalterMaier) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py]: should this be hardcoded

File: trunk/SUAVE/Components/Energy/Converters/__init__.py (hunk 5 above)
Comment (mclarke2) [reply trunk/SUAVE/Components/Energy/Converters/__init__.py]: Corrected, Issue Resolved.

File: trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py (hunk 4 above)
Comment (mclarke2) [reply trunk/SUAVE/Analyses/Aerodynamics/Vortex_Lattice.py]: Corrected, Issue Resolved.

File: regression/scripts/concorde/concorde.py (hunk 3 above)
Comment (mclarke2) [reply regression/scripts/concorde/concorde.py]: Corrected, Issue Resolved.

File: regression/scripts/B737/mission_B737.py (hunk 2 above)
Comment (mclarke2) [reply regression/scripts/B737/mission_B737.py]: Corrected, Issue Resolved.

File: regression/scripts/B737/mission_B737.py (hunk 1 above)
Comment (mclarke2) [reply regression/scripts/B737/mission_B737.py]: Corrected, Issue Resolved.

File: trunk/SUAVE/Components/Energy/Converters/__init__.py (hunk 5 above)
Comment (WalterMaier) [reply trunk/SUAVE/Components/Energy/Converters/__init__.py]: So Gearbox is not used?
No changes made

Comment (mclarke2) [reply trunk/SUAVE/Components/Energy/Converters/__init__.py]: it is used, I took it out by mistake

File: regression/scripts/AVL/test_AVL.py [hunk 7]
```diff
@@ -84,7 +84,7 @@ def main():
 
     # lift coefficient check
     lift_coefficient              = results.conditions.cruise.aerodynamics.lift_coefficient[0]
-    lift_coefficient_true         = 0.59495841
+    lift_coefficient_true         = 0.59324447
```
Comment (WalterMaier) [inline regression/scripts/AVL/test_AVL.py]: does this script use the .avl file in the regression?  I think we should include it since we upload it.

File: regression/scripts/Vehicles/Concorde.py [hunk 8]
```diff
@@ -138,7 +138,23 @@ def vehicle_setup(source_ratio=1.):
...
+    wing.Segments.append(segment)  
+    
+    # set tip section start point
+    segment = SUAVE.Components.Wings.Segment() 
+    segment.tag                   = 'tip'
+    segment.percent_span_location = 1.
+    segment.twist                 = 0. * Units.deg
+    segment.root_chord_percent    = 1.1/33.8
+    segment.dihedral_outboard     = 0.
+    segment.sweeps.quarter_chord  = 0. * Units.deg
```
Comment (WalterMaier) [inline regression/scripts/Vehicles/Concorde.py]: line these up

File: regression/scripts/concorde/concorde.py (hunk 3 above)
Comment (WalterMaier) [reply regression/scripts/concorde/concorde.py]: this still shows saving everytime

File: regression/scripts/concorde/concorde.py [hunk 9]
```diff
@@ -690,9 +690,40 @@ def check_results(new_results,old_results):
 def load_results():
     return SUAVE.Input_Output.SUAVE.load('results_mission_concorde.res')
 
+#def save_results(results):
+    #SUAVE.Input_Output.SUAVE.archive(results,'results_mission_concorde.res')
+    #return    
+    
 def save_results(results):
-    SUAVE.Input_Output.SUAVE.archive(results,'results_mission_concorde.res')
```
Comment (WalterMaier) [inline regression/scripts/concorde/concorde.py]: why is this removed/commented out?

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py [hunk 10]
```diff
@@ -62,69 +62,164 @@ def parasite_drag_wing(state,settings,geometry):
...
+        parasite_drag_coefficient = wing_parasite_drag ,
+        skin_friction_coefficient = (cf_w_u+cf_w_l)/2.   ,
+        compressibility_factor    = k_comp_u ,
+        reynolds_factor           = k_reyn_l , 
+        form_factor               = k_w    ,
+    )
     
-    # conditions
-    Mc  = freestream.mach_number
-    Tc  = freestream.temperature
```
Comment (WalterMaier) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py]: unneeded?

File: trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Drag/compressibility_drag_total.py [hunk 11]
```diff
@@ -215,18 +215,40 @@ def drag_div(Mc_ii,wing,k,cl,Sref_main):
 
         # Divergence mach number
         MDiv = np.array([[0.95]] * len(Mc_ii))
-        mcc = np.array([[0.93]] * len(Mc_ii))
+        mcc  = np.array([[0.93]] * len(Mc_ii))
 
     else:
         # Unpack wing
         t_c_w   = wing.thickness_to_chord
-        sweep_w = wing.sweeps.quarter_chord
```
Comment (WalterMaier) [inline trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Drag/compressibility_drag_total.py]: where did this go?

File: regression/scripts/Vehicles/Concorde.py (hunk 8 above)
Comment (mclarke2) [reply regression/scripts/Vehicles/Concorde.py]: Lined up: Issue resolved

File: regression/scripts/AVL/test_AVL.py (hunk 7 above)
Comment (mclarke2) [reply regression/scripts/AVL/test_AVL.py]: I changed the resolution. The AVL file is not created each time because it would require you to run AVL- which cannot be done on travis. I updated the results scripts which the regression is built upon

Comment (WallyMaier) [reply regression/scripts/AVL/test_AVL.py]: Gotcha, i think.  I think @timdmacdo knows more about this part than me.

File: regression/scripts/Vehicles/Concorde.py [hunk 12]
```diff
@@ -53,52 +53,52 @@ def vehicle_setup(source_ratio=1.):
     wing = SUAVE.Components.Wings.Wing()
```
Comment (planes) [inline regression/scripts/Vehicles/Concorde.py]: This has openvsp meshing associated with it. We need to make sure when it's run the CFD still converges if that's a desired quality. @timdmacdo  what do you think?

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/induced_drag_aircraft.py [hunk 13]
```diff
@@ -53,7 +53,7 @@ def induced_drag_aircraft(state,settings,geometry):
     e             = configuration.oswald_efficiency_factor
     K             = configuration.viscous_lift_dependent_drag_factor
     wing_e        = geometry.wings['main_wing'].span_efficiency
-    ar            = geometry.wings['main_wing'].aspect_ratio # TODO: get estimate from weissinger
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/induced_drag_aircraft.py]: I think the note was meant for span_efficiency. Maybe make a github issue

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py [hunk 14]
```diff
@@ -62,69 +62,164 @@ def parasite_drag_wing(state,settings,geometry):
...
-    t_c_w        = wing.thickness_to_chord
-    sweep_w      = wing.sweeps.quarter_chord
-    arw_w        = wing.aspect_ratio
-    span_w       = wing.spans.projected
-    S_exposed_w  = wing.areas.exposed # TODO: calculate by fuselage diameter (in Fidelity_Zero.initialize())
-    S_affected_w = wing.areas.affected  
-    xtu          = wing.transition_x_upper
-    xtl          = wing.transition_x_lower
+    # if wing has segments, compute and sum parasite drag of each segment
+    if len(wing.Segments.keys())>0:
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py]: set number of segments before the loop

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py [hunk 15]
```diff
@@ -62,69 +62,164 @@ def parasite_drag_wing(state,settings,geometry):
...
+        num_segments                 = len(wing.Segments.keys())     
+        total_wetted_area            = 0
+        total_segment_parasite_drag  = 0 
+        total_segment_k_w            = 0 
+        total_segment_cf_w_u         = 0
+        total_segment_cf_w_l         = 0 
+        total_segment_k_comp_u       = 0
+        total_k_reyn_l               = 0    
+        root_chord                   = wing.chords.root    
+        #Sref                         = 0
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py]: commented out

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py [hunk 16]
```diff
@@ -62,69 +62,164 @@ def parasite_drag_wing(state,settings,geometry):
...
+            wing.areas.wetted = total_wetted_area 
+            wing_parasite_drag= total_segment_parasite_drag  / Sref
+            k_w               = total_segment_k_w / Sref
+            cf_w_u            = total_segment_cf_w_u  / Sref
+            cf_w_l            = total_segment_cf_w_l / Sref
+            k_comp_u          =  total_segment_k_comp_u  / Sref
+            k_reyn_l          = total_k_reyn_l  / Sref
+
+    # if wing has no segments      
+    else:
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py]: I wonder if there is a way to make segmented and non segmented wings behave the same.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py [hunk 17]
```diff
@@ -62,69 +62,164 @@ def parasite_drag_wing(state,settings,geometry):
...
+                    taper         = chord_tip/chord_root   
+                    mac_seg       = chord_root * 2/3 * (( 1 + taper  + taper**2 )/( 1 + taper))
+                    Sref_seg      = span_seg*(chord_root+chord_tip)*0.5
+                    S_exposed_seg = Sref_seg
+ 
+                if wing.symmetric:
+                    Sref_seg = Sref_seg*2
+                    S_exposed_seg = S_exposed_seg*2
+                
+                #Sref += Sref_seg
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py]: commented out code

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py [hunk 18]
```diff
@@ -80,18 +82,88 @@ def weissinger_vortex_lattice(conditions,configuration,wing):
...
+            segment_chord = np.zeros(n_segments)
+            segment_twist = np.zeros(n_segments)
+            segment_sweep = np.zeros(n_segments)
+            segment_span = np.zeros(n_segments)
+            segment_chord_x_offset = np.zeros(n_segments)
+            section_stations = np.zeros(n_segments)
+            
+            # obtain chord and twist at the beginning/end of each segment
+            for i_seg in xrange(n_segments):                
+                segment_chord[i_seg] = wing.Segments[segment_keys[i_seg]].root_chord_percent*root_chord
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py]: you don't need to do [segment_keys[i_seg]] anymore, [i_seg] is sufficient

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py [hunk 19]
```diff
@@ -80,18 +82,88 @@ def weissinger_vortex_lattice(conditions,configuration,wing):
...
+            xa =  np.zeros(n)
+            x  = np.zeros(n)
+            y  =  np.zeros(n)
+            twist_distri =  np.zeros(n)
+            section_length =  np.zeros(n)
+            
+            # define coordinates of horseshoe vortices and control points
+            i_seg = 0
+            for idx in xrange(n):
+                twist_distri[idx]   =  segment_twist[i_seg] + ((yb[0][idx] - deltax[idx]/2 - section_stations[i_seg]) * (segment_twist[i_seg+1] - segment_twist[i_seg])/segment_span[i_seg+1])
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py]: I don't think this behaves as you expect.

Why did you calculate the all of the section properties but only use the first segment?

I think you meant to put this as a nested for loop but that is also highly frowned upon in SUAVE.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py [hunk 20]
```diff
@@ -80,18 +82,88 @@ def weissinger_vortex_lattice(conditions,configuration,wing):
...
+                
+                if i_seg == 0:
+                    segment_span[i_seg] = 0.0
+                    segment_chord_x_offset[i_seg] = 0.25*root_chord # weissinger uses quarter chord as reference
+                else:
+                    segment_span[i_seg]    = wing.Segments[segment_keys[i_seg]].percent_span_location*span - wing.Segments[segment_keys[i_seg-1]].percent_span_location*span
+                    segment_chord_x_offset[i_seg]  = segment_chord_x_offset[i_seg-1] + segment_span[i_seg]*np.tan(segment_sweep[i_seg-1])
+            
+            # shift spanwise vortices onto section breaks 
+            for i_seg in xrange(n_segments):
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py]: there's probably a way to do this without a for loop

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py [hunk 21]
```diff
@@ -80,18 +82,88 @@ def weissinger_vortex_lattice(conditions,configuration,wing):
...
+                idx =  (np.abs(y_coordinates-section_stations[i_seg])).argmin()
+                y_coordinates[idx] = section_stations[i_seg]
+            
+            # define y coordinates of horseshoe vortices      
+            ya = np.atleast_2d(y_coordinates[i])
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py]: tabulate this

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py [hunk 22]
```diff
@@ -117,13 +189,14 @@ def weissinger_vortex_lattice(conditions,configuration,wing):
     
         Cl = 2*LT/(0.5*Sref)
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py]: While we're here, make this CL and CD. These are 3D

File: trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Lift/vortex_lift.py [hunk 23]
```diff
@@ -0,0 +1,48 @@
+## @ingroup Methods-Aerodynamics-Fidelity_Zero-Lift
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Lift/vortex_lift.py]: Why are you duplicating this file?

This file exists in fidelity zero supersonic. If you need it too move it to common

File: trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Drag/compressibility_drag_total.py [hunk 24]
```diff
@@ -215,18 +215,40 @@ def drag_div(Mc_ii,wing,k,cl,Sref_main):
...
+        if len(wing.Segments.keys())>0: # if wing has segments
+            symm           = wing.symmetric
+            semispan       = wing.spans.projected*0.5 * (2 - symm)
+            root_chord     = wing.chords.root
+            num_segments   = len(wing.Segments.keys())     
+            
+            weighted_sweep = 0
+            Sref           = 0
+            for i_segs in xrange(num_segments): 
+                if i_segs == num_segments-1:
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Drag/compressibility_drag_total.py]: you know, you could make the loop one less...

File: trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Drag/compressibility_drag_total.py [hunk 25]
```diff
@@ -215,18 +215,40 @@ def drag_div(Mc_ii,wing,k,cl,Sref_main):
...
+                    continue 
+                else:                    
+                    span_seg        = semispan*(wing.Segments[i_segs+1].percent_span_location - wing.Segments[i_segs].percent_span_location )
```
Comment (planes) [inline trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Drag/compressibility_drag_total.py]: This is a higher level SUAVE question. But this could be made as a sizing function instead. Then it only needs to be called once. If we call it in the analysis we are calling this code hundreds of times in a mission.

Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Drag/compressibility_drag_total.py]: So we discussed that the compressibility drag correlation is for the full wing and it would be incorrect to perform this on individual segments then summing the drag. I corrected it in the subsonic fidelity zero compressibility drag computation but forgot to correct the supersonic comp. drag computation. Issue resolved

File: trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Drag/compressibility_drag_total.py (hunk 24 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Drag/compressibility_drag_total.py]: see comment below. Issue resolved

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py (hunk 22 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py]: Changed. Issue Resolved.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py (hunk 19 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py]: The twist distribution between each section is a  function of the starting segment twist plus a "delta twist" computer as a linear relationship between segments. As the span distance increases past a section, the segment number in increased (see line 141)

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py (hunk 18 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py]: Changed. Issue Resolved

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py (hunk 17 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py]: Removed. Issue Resolved

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py (hunk 16 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py]: Theoretically yes, defining two segments is the same as having defined no segments. However sometimes the use in SUAVE might not define segments. This code was made robust enough to handle either case.

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py (hunk 15 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py]: Removed. Issue Resolved

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py (hunk 14 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/parasite_drag_wing.py]: Changed. Issue Resolved

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/induced_drag_aircraft.py (hunk 13 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/induced_drag_aircraft.py]: Will do. Issue Resolved

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py (hunk 21 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py]: Tabulated. Issue Resolved

File: trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py (hunk 20 above)
Comment (mclarke2) [reply trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/weissinger_vortex_lattice.py]: Requires Loop. Issue resolved

**Comment by planes** [comment]
There's the same pylab hold error here too. But it looks like you have an incomplete merge too.

File: regression/scripts/Vehicles/Concorde.py [hunk 26]
```diff
@@ -62,34 +62,34 @@ def vehicle_setup():
     wing.taper                   = 0.
     wing.span_efficiency         = 0.9
     
-    wing.spans.projected         = 25.6    
+    wing.spans.projected           = 25.6
```
Comment (planes) [inline regression/scripts/Vehicles/Concorde.py]: why are all of these moved over?

File: regression/scripts/concorde/concorde.py [hunk 27]
```diff
@@ -737,7 +737,7 @@ def load_results():
 def save_results(results):
     SUAVE.Input_Output.SUAVE.archive(results,'results_mission_concorde.res')
     return    
-    
+
```
Comment (planes) [inline regression/scripts/concorde/concorde.py]: not a necessary change
